cumulusci/tests/test_utils.py

```diff
@@ -332,7 +332,7 @@ def test_package_xml_from_dict(self):
         )
 
 
-class TestTask(BaseTask):
+class FunTestTask(BaseTask):
```

Comment:
Is this just so it doesn't look like a test case? Sounds good to me. (And FUN!)

Reply:
Yup, I was playing with pytest and the warning was driving me nuts.